components/table/test/table.vdiff.js

```diff
@@ -27,6 +27,50 @@ function createSortableHeaderRow() {
 		</tr>
 	`;
 }
+
+function createSortableHeaderRowWithTwoButtons() {
+	return html`
+		<tr>
+			<th>
+				<d2l-table-col-sort-button>Double 1</d2l-table-col-sort-button>
+				<d2l-table-col-sort-button>Double 2</d2l-table-col-sort-button>
+			</th>
+			<th><d2l-table-col-sort-button desc>Cell A</d2l-table-col-sort-button></th>
+			<th><d2l-table-col-sort-button nosort>Cell B</d2l-table-col-sort-button></th>
+		</tr>
+	`;
+}
+
+function createSortableHeaderRowWithTwoElements() {
+	return html`
+		<tr>
+			<th>
+				<d2l-table-col-sort-button>Double 1</d2l-table-col-sort-button>
+				<div>Dummy text</div>
```

Comment:
I noticed that "Dummy text" is wrapping under the button. Is this because it's long or for another reason? Perhaps take a look at what's going on in the DOM in the context menu case in grades and add a visual diff for that case so that we can be sure that one will behave as expected.